pages/1.11/release-notes/1.11.7/index.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.7
+title: Release Notes for 1.11.7
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.7
+---
+
+DC/OS 1.11.7 was released on November 1, 2018. 
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+
+DC/OS 1.11.7 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/2ead30d/CHANGELOG).
+- Marathon 1.6.564 [change log](https://github.com/mesosphere/marathon/tree/3fa693b32).
+- Metronome 0.4.4 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.4).
+
+
+# Issues Fixed in DC/OS 1.11.7
+
+## GUI
+- DCOS-20222 - Add an error message when environment variables are input without keys. 
+- DCOS-42365 - Fix incorrect memory total in pod instance.
+- DCOS_OSS-1961 - Support lifecycle of pods in DC/OS UI.
+
+## Marathon
+- DCOS-43816 - Bump DC/OS OSS & EE to 1.11 for `marathon-1.6.564`.
+- DCOS_OSS-4193 - Marathon `systemd` unit requires zk-1.zk node to be available in order to launch Marathon service.
+
+## Mesos
+- COPS-3172/DCOS-40120/DCOS-42667 - Include MESOS-8545 and MESOS-9131 in Mesos bump.
```

Comment:
-  Fixed instances where parent containers may get stuck in `DESTROYING` state when there is a simultaneous health check running to create nested containers.